recipes/busco/run_test.sh

```diff
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+# test that find has the -delete flag, which BUSCO uses
+mkdir findtest ; cd findtest ; touch file ; find . -delete ; cd ..
```

Comment:
Why can this not be in the meta.yaml?

Reply:
This whole shell command? I originally had it in meta.yaml, but this is rather verbose and thus since there was already a run_test.sh, I moved it there.

Reply:
The current Docker tests, do not use external run_test files. So all test that you also want to run in Docker should be in meta.yaml.